deepcell_toolbox/utils.py

```diff
@@ -37,9 +37,10 @@
 from scipy.ndimage import fourier_shift
 from skimage.feature import register_translation
 from skimage import transform
-from skimage.segmentation import find_boundaries
-from skimage.morphology import remove_small_objects, square, dilation
-from skimage.measure import label
+from skimage.segmentation import find_boundaries, watershed
+from skimage.morphology import remove_small_objects, remove_small_holes
+from skimage.morphology import square, dilation
+from skimage.measure import label, regionprops
```

Comment:
Remove the unused imports

```suggestion
from skimage.measure import regionprops
from skimage.morphology import remove_small_holes
from skimage.segmentation import find_boundaries
```